src/components/Layout/Layout.tsx

```diff
@@ -0,0 +1,17 @@
+import React from "react"
+import Navbar from "components/Navbar"
+
+type Props = {
+  children: React.ReactNode
+}
+
+const Layout = (props: Props) => {
```

Comment:
```suggestion
const Layout: React.FC<Props> = (props) => {
```

Reply:
I'm not using react router. Next js built-in router is being used